bouncerscript/constants.py

```diff
@@ -90,9 +90,9 @@
 ]
 
 GO_BOUNCER_URL_TMPL = {
-    'project:releng:bouncer:server:production':
+    'production':
```

Comment:
It may be more precise to use `taskcluster_scope_prefix` rather than just split by `:`.  Then again, we’ve checked the server scope previously, so it may not be a blocker to do so.

Reply:
I wondered about just listing out the 4 scopes explicitly, happy to do that instead.

Reply:
Went ahead with the listing instead. I think the best fix for this in the long term would be to put the url for the public root in the puppet config, like we do for the [admin equivalent](https://github.com/mozilla-releng/build-puppet/blob/258a227faf172f9b98b264a8ff73abb083fe28db/modules/bouncer_scriptworker/manifests/settings.pp#L16).